docs/self-managed/setup/deploy/amazon/amazon-eks/terraform-setup.md

```diff
@@ -250,65 +227,419 @@ access_entries = {
+
+This configuration will deploy an OpenSearch domain with advanced security enabled. You must provide your own username (`advanced_security_master_user_name`) and password for the master user (`advanced_security_master_user_password`).
+
+</TabItem>
+</Tabs>
+
+#### Step 2: Additional customization
+
+You can further customize the OpenSearch cluster setup using various input options. For a full list of available parameters, see the [OpenSearch module documentation](https://github.com/camunda/camunda-tf-eks-module/blob/2.6.0/modules/opensearch/README.md).
+
+### Outputs
+
+**Terraform** allows you to define outputs, which make it easier to retrieve important values generated during execution, such as database endpoints and other necessary configurations for Helm setup.
+
+Each module that you have previously setup contains at the end the definition of the outputs. You can adjust them to your needs.
+
+These outputs will allow you to easily reference the **cert-manager** ARN, **external-dns** ARN, and the endpoints for both **PostgreSQL** and **OpenSearch** in subsequent steps or scripts, streamlining your deployment process.
+
+### Execution
+
+:::note Secret Management
+
+We strongly recommend managing sensitive information such as the OpenSearch, Aurora username and password using a secure secrets management solution like HashiCorp Vault. For details on how to inject secrets directly into Terraform via Vault, see the [Terraform Vault Secrets Injection Guide](https://developer.hashicorp.com/terraform/tutorials/secrets/secrets-vault).
+
+:::
+
+1. Open a terminal in the created Terraform folder where `config.tf` and other `.tf` files are.
+
+2. Plan the configuration files:
+
+```bash
+terraform plan -out cluster.plan # describe what will be created
+```
+
+3. After reviewing the plan, you can confirm and apply the changes.
+
+```bash
+terraform apply cluster.plan     # apply the creation
+```
+
+At this point, Terraform will create the Amazon EKS cluster with all the necessary configurations. The completion of this process may require approximately 20-30 minutes for each component.
+
+### Reference files
+
+Depending on the installation path you have chosen, you can find the reference files used on this page:
+
+- **Standard Installation:** [Reference Files](https://github.com/camunda/camunda-tf-eks-module/blob/feature/opensearch-doc/examples/camunda-8.6/)
+- **IRSA Installation:** [Reference Files](https://github.com/camunda/camunda-tf-eks-module/blob/feature/opensearch-doc/examples/camunda-8.6-irsa/)
+
+## 2. Preparation for Camunda 8 installation
+
+### Access the created EKS Cluster
+
+You can gain access to the Amazon EKS cluster via the `AWS CLI` using the following command:
 
 ```shell
-export CERT_MANAGER_IRSA_ARN=$(terraform output -raw cert_manager_arn)
+aws eks --region "$AWS_REGION" update-kubeconfig --name <clusterName>
```

Comment:
just a suggestion from my side to add `--alias <clusterName>`

so you end up with `<clusterName> in your k8s context instead of `arn:aws:eks:${AWS_REGION}:${ACCOUNT ID}:cluster/camunda-ci-eks`